transcripts/06-methods/4.txt

```diff
@@ -29,7 +29,7 @@
 1:31 oh look, in the subsequent sections on how Python deals with this,
 1:36 because this kind of flexibility is super powerful and Python does support it
 1:40 it just doesn't support it in the traditional method overloading way
-1:44 that you might be familiar with coming from C++, Java, C# and so on.
+1:44 that you might be familiar with coming from C++, Java, CSharp and so on.
```

Comment:
Ok, weird. Let's see how it comes out. If doesn't render correctly we should definitely switch this out. :)